classes/session/SessionManager.php

```diff
@@ -39,13 +39,16 @@ public function __construct(SessionDAO $sessionDao, PKPRequest $request)
         $this->sessionDao = $sessionDao;
 
         // Configure PHP session parameters
-        ini_set('session.use_trans_sid', 0);
-        ini_set('session.serialize_handler', 'php');
-        ini_set('session.use_cookies', 1);
-        ini_set('session.name', Config::getVar('general', 'session_cookie_name')); // Cookie name
+        ini_set('session.name', Config::getVar('general', 'session_cookie_name'));
         ini_set('session.cookie_lifetime', 0);
         ini_set('session.cookie_path', Config::getVar('general', 'session_cookie_path', $request->getBasePath() . '/'));
         ini_set('session.cookie_domain', $request->getServerHost(null, false));
+        ini_set('session.cookie_httponly', 1);
+        ini_set('session.cookie_samesite', Config::getVar('general', 'same_site', 'lax'));
+        ini_set('session.cookie_secure', Config::getVar('security', 'force_ssl'));
```

Comment:
This will clobber the proposal at https://github.com/pkp/pkp-lib/pull/8487. Can you work with @salmanm2003 to ensure that the two are reconciled?

Reply:
@asmecher I have already added these two lines for the next review as it will make sure that the cookies will have secure flag and http_only enabled before and after login as being mentioned at #7373 by @fradeve :
```
ini_set('session.cookie_httponly', 1);
ini_set('session.cookie_secure', Config::getVar('security', 'force_ssl'));

Reply:
Yep! For now the #8487 is updating only the line that I dropped, but as @salmanm2003 mentioned he will add the same code, then things will probably end looking the same...

Well, I'm not going to update anything more related to the "secure/httponly" flags, so I'll leave it this way for now, then I might incorporate his PR or vice-versa :)